app/scripts/controllers/decrypt-message.ts

```diff
@@ -35,6 +36,27 @@ const stateMetadata = {
   unapprovedDecryptMsgCount: { persist: false, anonymous: false },
 };
 
+export const isEIP1024EncryptedMessage = (message: {
```

Comment:
Nit: Adding a JSDoc comment here would be great! Our normal lint rules require it for all declared functions